client/webserver/site/src/js/wallets.ts

```diff
@@ -1086,11 +1089,28 @@ export default class WalletsPage extends BasePage {
     this.updateDisplayedReconfigFields(currentDef)
   }
 
+  showOrHideRecoverySupportMsg (wallet:WalletState, seeded: boolean) {
+    const page = this.page
+    if (seeded && !wallet.running && !wallet.disabled && wallet.traits & traitRecoverer) {
+      page.reconfigSupportMsg.textContent = intl.prep(intl.ID_WALLET_RECOVERY_SUPPORT_MSG, { walletSymbol: wallet.symbol.toLocaleUpperCase() })
+      Doc.show(page.reconfigSupportMsg)
+      page.submitReconfig.setAttribute('disabled', '')
+      page.submitReconfig.classList.add('grey')
+      return
+    }
+    page.submitReconfig.removeAttribute('disabled')
+    page.submitReconfig.classList.remove('grey')
+    Doc.empty(page.reconfigSupportMsg)
+    Doc.hide(page.reconfigSupportMsg)
+  }
+
   changeWalletType () {
     const page = this.page
     const walletType = page.changeWalletTypeSelect.value || ''
     const walletDef = app().walletDefinition(this.selectedAssetID, walletType)
     this.reconfigForm.update(this.selectedAssetID, walletDef.configopts || [], false)
+    const wallet = app().walletMap[this.selectedAssetID]
+    this.showOrHideRecoverySupportMsg(wallet, walletDef.seeded)
```

Comment:
When I have a borked native wallet and I change the wallet type to RPC, the recovery support message is correctly hidden, but that's only because the rpc wallet isn't seeded. If we, for instance, had two seeded wallet types, and the user tried to change to the other type, the message would still show and the user would not be able to submit. The logic should be more like this https://github.com/buck54321/dcrdex/compare/ccc0c7e...buck54321:dcrdex:recovery-viz